tools/debian.md

```diff
@@ -9,6 +9,11 @@ releaseColumn: true
 releaseDateColumn: true
 sortReleasesBy: 'release'
 releases:
+  - releaseCycle: 'Debian 11 "Bullseye"'
+    release: 2021-08-15
+    eol: 2026-08-15
```

Comment:
I'll note https://wiki.debian.org/DebianReleases doesn't list an official EOL yet, as it depends on when the _next_ release is out.

> Reminder: the EOL date for the stable release is the date of the next stable release plus one year. For example, Debian bullseye ("buster +1") was released on 2021-08-14 so Debian buster's EOL is around 2022-08.

Reply:
I'm basing this off the release announcement: https://www.debian.org/News/2021/20210814

> Debian project is proud to present its new stable version 11 (code name "bullseye"), which will be supported for the next 5 years

I believe this is good enough to at least provide an estimate for now that we can always update later down the road.

Reply:
Yep, sounds reasonable!